pkgs/by-name/c-/c-periphery/package.nix

```
@@ -0,0 +1,29 @@
+{ lib, stdenv, fetchFromGitHub, cmake }:
```

nit: this would make the code a bit more diff friendly

```suggestion
{ lib
, stdenv
, fetchFromGitHub
, cmake
}:
```